tests/sentry/snuba/test_discover.py

```diff
@@ -570,9 +570,60 @@ def test_reflective_types(self):
         assert results["meta"] == {
             "p50_measurements_lcp": "duration",
             "p50_measurements_foo": "number",
-            "p50_span_op_breakdowns_foo": "duration",
+            "p50_spans_foo": "duration",
         }
 
+    def test_measurements(self):
+        event_data = load_data("transaction", timestamp=before_now(seconds=3))
+        self.store_event(data=event_data, project_id=self.project.id)
+
+        results = discover.query(
+            selected_columns=[
+                "measurements.fp",
+                "measurements.fcp",
+                "measurements.lcp",
+                "measurements.fid",
+                "measurements.cls",
+                "measurements.does_not_exist",
+            ],
+            query="event.type:transaction",
+            params={"project_id": [self.project.id]},
+        )
+
+        data = results["data"]
+        assert len(data) == 1
+        assert data[0]["measurements.fp"] == event_data["measurements"]["fp"]["value"]
+        assert data[0]["measurements.fcp"] == event_data["measurements"]["fcp"]["value"]
+        assert data[0]["measurements.lcp"] == event_data["measurements"]["lcp"]["value"]
+        assert data[0]["measurements.fid"] == event_data["measurements"]["fid"]["value"]
+        assert data[0]["measurements.cls"] == event_data["measurements"]["cls"]["value"]
+        assert data[0]["measurements.does_not_exist"] is None
+
+    def test_span_op_breakdowns(self):
+        event_data = load_data("transaction", timestamp=before_now(seconds=3))
+        self.store_event(data=event_data, project_id=self.project.id)
+
+        results = discover.query(
+            selected_columns=[
+                "spans.http",
+                "spans.db",
+                "spans.resource",
+                "spans.browser",
+                "spans.does_not_exist",
```

Comment:
Why is it `spans.http` here and `spans.ops.http` everywhere else?

Reply:
@Zylphrex It was a find and replace typo on my part. Should be fixed in https://github.com/getsentry/sentry/pull/25168/commits/995a6f38e29721366332fa64f8837060a1f051c1

Only fields that begin with `spans.` are matched. Technically `spans.ops.http` is legal and would resolve to the field `span_op_breakdowns[ops.ops.http]`. It was a good thing I added this query integration test :)